addons/zylann.scatter/tools/plugin.gd

```diff
@@ -95,6 +91,8 @@ func make_visible(visible):
 func forward_spatial_gui_input(p_camera, p_event):
 	if _node == null:
 		return false
+	if _selected_patterns.empty():
+		return false
```

Comment:
Adding this will prevent deleting patterns (or doing other actions) when palette selection is empty. I think it would only be relevant for painting. Although, I guess you added it because now `edit()` no longer selects an item by default, so "empty selection" became a possible state.

Reply:
I moved the check to where its relevant, allowing erasing with no patterns selected.